apps/authentication-service/src/resources/clients/clientsRoutesV1.ts

```diff
@@ -0,0 +1,76 @@
+import { Client } from "auth0";
+import { Request, Response, Router } from "express";
+
+import { getClientTokenFromRequest } from "@/shared/utils";
+import { validateAccessToken } from "@/middleware/authMiddleware";
+import {
+  getClient,
+  createClient,
+  deleteClient,
+} from "@/resources/clients/clientsService";
+import { ClientCreateBody } from "@/resources/clients/clientsModel";
+
+const apiVersion = "v1";
+
+export const clientsRoutesV1 = (router: Router): void => {
+  router.post(
+    "/",
+    [validateAccessToken],
+    (req: Request<object, object, ClientCreateBody>, res: Response) => {
```

Comment:
All the handlers in this file need to be exported and tested

Reply:
I'm not sure I understand. What is in this file that we're not already testing in the e2e tests?

Reply:
This blurb from our testing ADR does a decent job of explaining what we're after:

"We will mock as little as possible in our tests and [prefer integration tests over unit tests](https://kentcdodds.com/blog/write-tests). The bulk of our tests will be integration tests, because they provide the best performance to confidence ratio.

The purpose of our end to end tests will be to validate that the frontend is working with the backend properly, and that apis are working together properly. We don't need to test every edge case in our end to end tests, because using MSW allows us to test those edge cases in our integration tests.

The purpose of our tests is to give us confidence to make changes without breaking things. We will write enough tests that we can be confident in deploying changes without manual testing."

IMO E2E tests are extra insurance. Everything that can be tested using integration tests should be tested that way. If an E2E test gives us extra confidence, then we write it. E2E tests are great for validating that the things we're mocking in our integration tests work when using the real thing.

Reply:
Ah, okay. I see. Thanks for pointing that out.